### PR TITLE
Closes #16

### LINKED ISSUE #16: Update README

### PR DESCRIPTION
**Pull request by gorbuvla**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
 [![Build Status](https://travis-ci.org/AckeeCZ/ackroutine-adapter.svg?branch=master)](https://travis-ci.org/AckeeCZ/ackroutine-adapter) [ ![Download](https://api.bintray.com/packages/ackeecz/ackroutine-adapter/coroutine-adapter/images/download.svg) ](https://bintray.com/ackeecz/ackroutine-adapter/coroutine-adapter/_latestVersion)
 
 # Coroutine OAuth Android Library
-Simple coroutine extension, that adds support to Retrofit2 based projects which uses OAuth2 authentication.
+Simple coroutine extension, which makes use of Retrofit2 internal `Call<T>` to suspending function conversion to add support for 
+OAuth and custom request modifiers.
```
Comment (davidbilik) [inline README.md]: I think that its not correct to wrap a line like that. It should be all in a single line and browser will decide where to wrap it

**Comment by davidbilik** [comment]
I think that you can add to description something like `Closes #16`

**Comment by davidbilik** [comment]
The same applies to commits :) 
```
Related: #16
```

Lets make our opensource great :)

File: README.md (hunk 1 above)
Comment (gorbuvla) [reply README.md]: ✅

**Pull request merged**